aries_cloudagent/protocols/actionmenu/v1_0/controller.py

```diff
@@ -19,3 +19,7 @@ async def determine_roles(self, context: InjectionContext) -> Sequence[str]:
         service = context.inject_or(BaseMenuService)
         if service:
             return ["provider"]
+
+    def determine_goal_codes(self) -> Sequence[str]:
+        """Return defined goal_codes."""
+        return []
```

Comment:
In the context of `GoalCodeRepository`, I am using `controller` to play a similar role to `message_types` (with ProtocolsRepository), `determine_goal_code` is returning the specified `goal-codes` for that protocol to build up the repository and then `goal_codes_matching_query` is called upon in there.

May be, I can rename `determine_goal_codes` to `return_goal_codes` to avoid any confusion?

Reply:
@shaangill025 it's fine you don't need to change it, PR is approved :-)